implementations/rust/ockam/ockam_api/src/lib.rs

```diff
@@ -22,6 +23,7 @@ pub mod lmdb;
 
 #[macro_use]
 extern crate tracing;
+extern crate core;
```

Comment:
Could you please elaborate what is the purpose of that?

Reply:
shouldn't be there, and it's not needed

Reply:
removing